scripts/un/sdg/README.md

```diff
@@ -39,4 +40,15 @@ python3 -m unittest discover -v -s ../ -p "*_test.py"
 ```
 
 Notes:
-* We currently drop certain series and variables (refer to `util.py` for the list) which have been identified by UN as potentially containing outliers.
\ No newline at end of file
+* We currently drop certain series and variables (refer to `util.py` for the list) which have been identified by UN as potentially containing outliers.
+
+### SDMX
+
+As reference, we provide an earlier version of the import scripts that utilized the UN API (which uses SDMX) in the `sdmx/` folder. Please note that these scripts may have errors and do not use the most up-to-date schema format, so should only be used as an illustration of the SDMX -> MCF mapping and should not actually be run.
```

Comment:
Suggestion: Bold "should not be run" or wrap the warning in a quote callout

Reply:
Also, consider adding "This script does not use the most up-to-date schema format. It should only be used as an illustration of the SDMX -> MCF mapping. Do not actually run" to the top of the relevant .py script files.

Reply:
done